cve_bin_tool/checkers/apache_http_server.py

```diff
@@ -0,0 +1,23 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+
+"""
+CVE checker for Apache HTTP Server
+
+https://www.cvedetails.com/vulnerability-list/vendor_id-45/product_id-66/Apache-Http-Server.html
+"""
+from cve_bin_tool.checkers import Checker
+
+
+class ApacheHttpServerChecker(Checker):
+    CONTAINS_PATTERNS = [
+        r"logs/apache_runtime_status",
+        r"httpd not running, trying to start",
+        r"-D HTTPD_ROOT=\"/etc/httpd\"",
+    ]
+    FILENAME_PATTTERN = [r"httpd"]
+    VERSION_PATTERNS = [
+        r"Apache/([0-9]+\.[0-9]+(\.[0-9]+)?)",
```

Comment:
This seems like it might be a problematic pattern, since apache is a pretty large foundation.  Can you check a few of their other products and make sure this is only used in httpd?

Reply:
Okay, I'll try to do that, however the problem is that this seemed to be the only line that had the version number displayed cleanly in the binary.